istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -600,38 +633,24 @@ func StatsConfigCmd(ctx cli.Context) *cobra.Command {
 			return nil
 		},
 		RunE: func(c *cobra.Command, args []string) error {
-			var stats string
 			kubeClient, err := ctx.CLIClient()
 			if err != nil {
 				return err
 			}
-			if podName, podNamespace, err = getPodName(ctx, args[0]); err != nil {
-				return err
-			}
-			if statsType == "" || statsType == "server" {
-				stats, err = setupEnvoyServerStatsConfig(kubeClient, podName, podNamespace, outputFormat)
-				if err != nil {
+			if labelSelector != "" {
```

Comment:
What will happen if I type both arg and labelSelector flag? Do we need to improve the args validation function?

Reply:
Thanks, there are several ways:

1. Label selector first (this pr)
2. podname first
3. Issue a warning and exit, like kubectl: `error: name cannot be provided when a selector is specified`
4. Display both (I have never seen this method)

Which one do you think is more appropriate?

Reply:
We should have a similar situation in other commands, and it’s better to be consistent with them. Can you check?

Reply:
Let's use the label first logic. We can do similar arg verification like https://github.com/istio/istio/blob/68cb67731f1f6cd29e38c22e38ef0b7d48512fcd/istioctl/pkg/dashboard/dashboard.go#L239-L242